synfig-core/src/synfig/savecanvas.cpp

```diff
@@ -579,10 +579,16 @@ xmlpp::Element* encode_dynamic_list(xmlpp::Element* root,ValueNode_DynamicList::
 	ValueNode_WPList::ConstHandle wplist_value_node(ValueNode_WPList::ConstHandle::cast_dynamic(value_node));
 	ValueNode_DIList::ConstHandle dilist_value_node(ValueNode_DIList::ConstHandle::cast_dynamic(value_node));
 
+	bool must_rotate_point_list = false;
+
 	if(bline_value_node)
 	{
 		if(bline_value_node->get_loop())
+		{
 			root->set_attribute("loop","true");
+			if (save_canvas_version < RELEASE_VERSION_1_4_0) // or get_file_version()?
```

Comment:
@blackwarthog could you point me the difference between `save_canvas_version` and `get_file_version()` ?
Can they store different values or are they always the same?

Reply:
`get_file_version()` returns `save_canvas_version`, so I think they are always the same.